src/intree.js

```diff
@@ -8,6 +8,15 @@ const jsone = require('json-e');
 
 // Assert that only scope-valid characters are in branches
 const branchTest = /^[\x20-\x7e]*$/;
+let slugids = {};
```

Comment:
This needs to be scoped to the run of the intree function -- putting it here scopes it to the lifetime of the tc-github process, which is too long :)